src/shim/sdk/DocumentSDK.ts

```diff
@@ -112,17 +112,17 @@ export function decryptFromStore(documentID: string) {
 /**
  * Decrypt the provided document given the ID of the document and its data. Returns a Promise which will be resolved once the document has been successfully decrypted.
  * @param {string}      documentID   Unique ID of document to decrypt
- * @param {Base64Bytes} documentData Document data to decrypt
+ * @param {Base64Bytes | Uint8Array} documentData Document data to decrypt
  */
-export function decrypt(documentID: string, documentData: Base64Bytes) {
+export function decrypt(documentID: string, documentData: Base64Bytes | Uint8Array) {
```

Comment:
We also need to update the `https://github.com/IronCoreLabs/ironweb/blob/master/ironweb.d.ts` file so the proper types are published.